package.json

```diff
@@ -15,8 +15,13 @@
     "semver": "^5.3.0"
   },
   "devDependencies": {
+    "@types/chai": "^4.0.1",
     "@types/lodash.memoize": "^4.1.2",
+    "@types/mocha": "^2.2.41",
     "@types/semver": "^5.3.32",
```

Comment:
I've created an issue for this https://github.com/resin-io-modules/resin-semver/issues/8